x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -69,16 +70,28 @@ export function getDefaultCapabilities(): MlCapabilities {
 export function getPluginPrivileges() {
   const userMlCapabilitiesKeys = Object.keys(userMlCapabilities);
   const adminMlCapabilitiesKeys = Object.keys(adminMlCapabilities);
-  const allMlCapabilities = [...adminMlCapabilitiesKeys, ...userMlCapabilitiesKeys];
+  const allMlCapabilitiesKeys = [...adminMlCapabilitiesKeys, ...userMlCapabilitiesKeys];
+
+  const privilege = {
+    app: [PLUGIN_ID, 'kibana'],
+    excludeFromBasePrivileges: true,
+    catalogue: [PLUGIN_ID],
```

Comment:
We should declare ML's management app as part of this feature as well:

```suggestion
    management: {
      insightsAndAlerting: ['jobsListLink`]
    },
    catalogue: [PLUGIN_ID],
```